tools/rna_tools/dot2ct/datatypes_conf.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0"?>
+<datatypes>
```

Comment:
For more standards please refer to: http://galaxy-iuc-standards.readthedocs.org/en/latest/